.github/minimal-rmf/Dockerfile

```diff
@@ -31,3 +16,8 @@ ENV PATH "$PNPM_HOME:$PATH"
 RUN pnpm env use --global lts
 
 RUN apt update && apt install -y python3-venv
+
+# cleanup
+RUN rm -rf \
+    /var/lib/apt/lists \
+    /dist
```

Comment:
Is `/dist` used by any commands?

Reply:
https://github.com/open-rmf/rmf-web/pull/1060/commits/e794621373f4e907a9c2ecaf37eacd87da329b0f